src/domain/siwe/siwe.repository.ts

```diff
@@ -75,105 +55,4 @@ export class SiweRepository implements ISiweRepository {
-   *
-   * @param args.message - SiWe message
-   * @param args.signature - signature from signing {@link args.message}
-   *
-   * @returns boolean - whether the signed message is valid
-   */
-  async isValidMessage(args: {
-    message: string;
-    signature: `0x${string}`;
-  }): Promise<boolean> {
-    const message = parseSiweMessage(args.message);
-
-    // Without nonce we can't verify whether a replay attack
-    if (!message.nonce) {
-      return false;
-    }
-
-    try {
-      // Verifying message after notBefore and before expirationTime
-      const isValidMessage = validateSiweMessage({
-        message,
-      });
-
-      if (!isValidMessage || !message.chainId || !message.address) {
-        return false;
-      }
-
-      // Expiration expectation does not exceed max validity period
-      const isExpirationValid =
-        !message.expirationTime ||
-        message.expirationTime <=
-          new Date(Date.now() + this.maxValidityPeriodInSeconds * 1_000);
-
-      if (!isExpirationValid) {
-        return false;
-      }
-
-      // Verify signature and nonce is cached (not a replay attack)
-      const [isValidSignature, isNonceCached] = await Promise.all([
-        this.isValidSignature({
-          chainId: message.chainId.toString(),
-          address: message.address,
-          message: args.message,
-          signature: args.signature,
-        }),
-        this.siweApi.getNonce(message.nonce).then(Boolean),
-      ]);
-
-      return isValidSignature && isNonceCached;
-    } catch (e) {
-      this.loggingService.debug(
-        `Failed to verify SiWe message. message=${args.message}, error=${e}`,
-      );
-      return false;
-    } finally {
-      await this.siweApi.clearNonce(message.nonce);
-    }
-  }
-
-  getMaxValidityDate(): Date {
-    return new Date(Date.now() + this.maxValidityPeriodInSeconds * 1_000);
-  }
-
-  /**
-   * Verifies signature of signed SiWe message, either by EOA or smart contract
-   *
-   * @param args.message - SiWe message
-   * @param args.chainId - chainId of the blockchain
-   * @param args.address - address of the signer
-   * @param args.signature - signature from signing {@link args.message}
-   *
-   * @returns boolean - whether the signature is valid
-   */
-  private async isValidSignature(args: {
-    message: string;
-    chainId: string;
-    address: `0x${string}`;
-    signature: `0x${string}`;
-  }): Promise<boolean> {
-    // First check if valid signature of EOA as it can be done off chain
-    const isValidEoaSignature = await verifyMessage(args).catch(() => false);
-    if (isValidEoaSignature) {
-      return true;
-    }
-
-    // Else, verify hash on-chain using ERC-6492 for smart contract accounts
-    const blockchainApi = await this.blockchainApiManager.getApi(args.chainId);
-    return blockchainApi.verifySiweMessage({
-      message: args.message,
-      signature: args.signature,
-    });
```

Comment:
This was not being used, not would work with Safe signatures in its current form.